src/ts/content/extensions/util/onLogged.ts

```diff
@@ -8,8 +8,7 @@ import {onBackgroundEvent} from "../../util/onBackgroundEvent";
 import {isAuthorized} from "../author/util/isAuthorized";
 import {isSheetSet} from "../../../common/entities/spreadsheet";
 
-const BAD_BROWSER_INFO_URL =
-	"https://github.com/braxtonhall/library-thing/blob/main/docs/librarian/authors.md#prerequisites";
+const BAD_BROWSER_INFO_URL = "https://betterlibrarything.com/";
```

Comment:
shouldn't this link be https://betterlibrarything.com/docs/librarian/authors.md.pretty

Reply:
No I think it should just go to the main page, which shows the list of compatible browsers.

Reply:
o ok fair